src/docs/03_system_scope_and_context.adoc

```diff
@@ -23,44 +23,63 @@ Various options:
 
 === Business Context
 
-[role="arc42help"]
-****
-.Contents
-Specification of *all* communication partners (users, IT-systems, ...) with explanations of domain specific inputs and outputs or interfaces.
-Optionally you can add domain specific formats or communication protocols.
 
-.Motivation
-All stakeholders should understand which data are exchanged with the environment of the system.
 
-.Form
-All kinds of diagrams that show the system as a black box and specify the domain interfaces to communication partners.
+image::SystemScopeDiagram.png[System scope diagram] 
 
-Alternatively (or additionally) you can use a table.
-The title of the table is the name of your system, the three columns contain the name of the communication partner, the inputs, and the outputs.
-****
 
-**<Diagram or Table>**
+.Viade
+|=========================================================
+| Communication partner |Inputs |Outputs
+
+| User
+| Upload routes. Attaches photos, videos or information to them
+| Reads routes from its own pod
+
+| Pod
+| Can store routes in a interoperable route format
+| Allow viade to read the routes stored inside
+
+| Viade
+| Routes and multimedia 
+| Routes and push notifications
+
+| User's contact
+| 
+| Receives push notifications when a route is shared
+
+|=========================================================
+
 
-**<optionally: Explanation of external domain interfaces>**
 
 === Technical Context
 
-[role="arc42help"]
-****
-.Contents
-Technical interfaces (channels and transmission media) linking your system to its environment. In addition a mapping of domain specific input/output to the channels, i.e. an explanation with I/O uses which channel.
+image::technicalcontext.jpg[Technical context diagram] 
```

Comment:
Same as on the first image, it is not shown, there is no image on the images file.

Reply:
Check the commits, two images are uploaded in the corresponding folder